### DISCUSSION TITLE
github profile with Dev Card

### DISCUSSION THREAD

**Discussion by Luis981123** [DenverCoder1/github-readme-streak-stats#652] [General]
Do you know about dev card?

**Comment by Luis981123** [comment]
Hi guys.
If you know it, Please let me know

**Reply by Arturs1123** [reply]
yeah, I know it.
Please check the following url:
https://daily.dev/blog/adding-the-daily-devcard-to-your-github-profile?utm_source=webapp&utm_medium=devcard&utm_campaign=devcardguide&utm_id=inapp
